Assets/Scripts/Level/TutorialTriggerGroup.cs

```diff
@@ -30,6 +33,10 @@ void UpdateTriggers()
         }
         playerProgress = GameObject.FindWithTag(PlayerConstants.PlayerTag).GetComponent<PlayerProgress>();
         gameUI = playerProgress.GetComponentInChildren<InGameUI>(true);
+        if (playerProgress == null || gameUI == null)
+        {
+            Debug.LogError("Unable to find playerProgress or gameUI");

```

Comment:
probably add a return to prevent uncaught nullref below

Reply:
It doesn't change anything since null is a valid value for those elements so the rest of the code functions fine. But sure. It'll save a few CPU cycles.